Tests/CommandsTests/BuildToolTests.swift

```diff
@@ -71,16 +71,18 @@ final class BuildToolTests: CommandsTestCase {
     }
 
     func testImportOfMissedDepWarning() throws {
+        try XCTSkipIf(true, "tested functionality doesn't work due to 'libSwiftScan dependency scan query failed' (rdar://108512158)")
```

Comment:
should it be conditionally disabled? or disabled everywhere?

Reply:
conditional would be nice, but I'm not aware of any way to determine the conditional right now.